scripts/gen_server_keys.sh

```diff
@@ -18,19 +18,20 @@ COMMONNAME=""
 ADDRESS=""
 SUBJECT="/C=US"
 
-while getopts d:a: option; do
+while getopts d:a:s: option; do
```

Comment:
There already is a (merged) PR for that: https://github.com/google/keytransparency/pull/725
Will clean it in this branch. Sorry for the noise.